decidim-budgets/app/views/decidim/budgets/budgets/index.html.erb

```diff
@@ -1,5 +1,28 @@
-<%= render partial: "decidim/shared/component_announcement" %>
+<% content_for :js_content do %>
+  <%= javascript_pack_tag "decidim_budgets" %>
+<% end %>
 
-<%= cell("decidim/budgets/budgets_header", current_workflow) %>
+<% content_for :css_content do %>
+  <%= stylesheet_pack_tag "decidim_budgets", media: "all" %>
+<% end %>
 
-<%= cell("decidim/budgets/budgets_list", current_workflow) %>
+<% content_for :aside do %>
+  <h1 class="h1 decorator">
+    <%= t("name", scope: "decidim.components.budgets") %>
+  </h1>
+<% end %>
+
+<%= render layout: "layouts/decidim/shared/layout_two_col" do %>
+
+  <% announcement_body = if translated_attribute(current_settings.announcement).present?
```

Comment:
I'm going to move the announcement_body to a helper method and remove the `if announcement_body.present?` in the call to the cell, because the cell doesn't render nothing if receives a blank argument